pkg/component/observability/monitoring/prometheus/longterm/scrapeconfigs.go

```diff
@@ -61,6 +61,7 @@ func CentralScrapeConfigs() []*monitoringv1alpha1.ScrapeConfig {
 						`{__name__="garden_seed_usage"}`,
 						`{__name__="garden_seed_capacity"}`,
 						`{__name__="etcdbr_snapshot_duration_seconds_count"}`,
+						`{__name__="apiserver_request_total", job=~="(virtual-garden-kube-apiserver)"}`
```

Comment:
```suggestion
						`{__name__="apiserver_request_total", job=~="(virtual-garden-kube-apiserver)"}`,
```

Reply:
Shouldn't it be?
```suggestion
						`{__name__="apiserver_request_total", job=~"(virtual-garden-kube-apiserver)"}`,
```

In general I'd vote for:
```suggestion
						`{__name__="apiserver_request_total", job="virtual-garden-kube-apiserver"}`,
```

Reply:
oh yes 😅